app/presenters/activity_csv_presenter.rb

```diff
@@ -3,4 +3,9 @@ def intended_beneficiaries
     return if super.blank?
     to_model.intended_beneficiaries.map { |item| I18n.t("activity.recipient_country.#{item}") }.join("; ")
   end
+
+  def beis_id
+    return "" if super.blank?
```

Comment:
Do we need to do this? `nil.to_s` is functionally the same as doing this

Reply:
🤦‍♀️ yes, thanks for pointing this out! 😄